### PR TITLE
Horiz brace

### PR DESCRIPTION
**Pull request by marcianx**
1) Copy-paste.
2) Add flow types.

Man, I keep learning new things about this code base (and the inadequacy of some of the types, which I'm going to have to deal with later) as I keep adding strict type checking.

### CODE REVIEW

File: src/functions.js [hunk 1]
```diff
@@ -73,19 +54,7 @@ import "./functions/font";
...
-}, function(context, args) {
-    const base = args[0];
-    return {
-        type: "horizBrace",
-        label: context.funcName,
-        isOver: /^\\over/.test(context.funcName),
-        base: base,
-    };
-});
+import "./functions/horizBrace";
```
Comment (ylemkimon) [inline src/functions.js]: 👍
I think we can remove new lines between `import` statements as they are single line.

File: src/functions.js [hunk 2]
```diff
@@ -1,31 +1,12 @@
...
-import {
-    default as _defineFunction,
-    _functions,
-} from "./defineFunction";
-
-import type {FunctionPropSpec, FunctionHandler} from "./defineFunction";
-import type {NodeType} from "./ParseNode";
+import {_functions} from "./defineFunction";
 
 // WARNING: New functions should be added to src/functions and imported here.
```
Comment (ylemkimon) [inline src/functions.js]: I think we can remove the warning, too.

Comment (marcianx) [reply src/functions.js]: Done.

File: src/functions.js (hunk 1 above)
Comment (marcianx) [reply src/functions.js]: Done.
Also, as an additional commit at the end, I remove intermediate comments (these appear inconsistently only on an arbitrary subset) and alphabetized the whole list of functions/ imports. I can undo this last part if you prefer.

**Pull request merged**